conbench/templates/compare-entity.html

```diff
@@ -1,5 +1,40 @@
 {% extends "app.html" %}
 
+{# This macro displays Tags, Hardware, Context and Info fields in comparative view so
+matching fields between baseline and contender are aligned and unmatching fields and/or values are highligted in red #}
```

Comment:
```suggestion
matching fields between baseline and contender are aligned and mismatched fields and/or values are highlighted in red #}
```

Very very minor language tweak